src/libraries/restore/Directory.Build.props

```diff
@@ -7,11 +7,8 @@
     <ProjectAssetsFile>$(RestoreOutputPath)/project.assets.json</ProjectAssetsFile>
     <MSBuildProjectExtensionsPath>$(IntermediateOutputPath)</MSBuildProjectExtensionsPath>
     <SkipDeriveTargetFrameworks>true</SkipDeriveTargetFrameworks>
-    <!-- let us control the output path -->
-    <OutputPath>$(BaseOutputPath)$(TargetFramework)-$(TargetFrameworkSuffix)-$(Configuration)</OutputPath>
-    <OutputPath Condition="'$(TargetFrameworkSuffix)' == ''">$(BaseOutputPath)$(TargetFramework)-$(Configuration)</OutputPath>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-    <AdditionalBuildTargetFrameworks Condition="'$(BuildAllProjects)' == 'true'">netstandard2.0-$(TargetFrameworkSuffix)</AdditionalBuildTargetFrameworks>
+    <AdditionalBuildTargetFrameworks Condition="'$(BuildAllProjects)' == 'true'">netstandard2.0</AdditionalBuildTargetFrameworks>
```

Comment:
Why are you removing the TargetFrameworkSuffix here?

Reply:
What about `RestoreOutputPath`, `ProjectAssetsFile` and `MSBuildProjectExtensionsPath` above, which all derive from `IntermediateOutputPath` which derives from `TargetFramework`.

Reply:
As eric mentioned , the next step is to remove the usage of all properties in targetFramework.props from propertyGroups.
eg all these properties will be the candidate for that.

Reply:
> Why are you removing the TargetFrameworkSuffix here?

This was a bug. TargetFrameworkSuffix is a property for innerBuilds and is not defined for outer builds and should not be used in outer build. AdditionalBuildTargetFrameworks is used only in outerbuilds.

Reply:
I'm unsure what's the scope of this PR. Isn't the goal to fix the broken intermediate/output paths? An example:

![image](https://user-images.githubusercontent.com/7412651/78370528-0d483880-75c7-11ea-9baf-f5b1667058fc.png)

Reply:
The scope is just cleanup, nothing is getting fixed here.